File: CloudComp/1_21_22/Point.py
from math import hypot
class Point:
    def __init__(self, x = 0, y = 0):
        self.x = x
        self.y = y
    def __str__(self):
        return f"({self.x}, {self.y})"
    def distance_to(self, q):
        return hypot(self.x - q.x, self.y - q.y)
p = Point(3,4)
q = Point()
print(p.distance_to(q))

File: CloudComp/1_21_22/Point.java
public class Point {
    private int x;
    private int y;
    public Point(int x, int y){
        this.x = x;
        this.y = y;
    }
    public Point(){
        this(0,0);
    }
    @Override
    public String toString(){
        return String.format("(%s, %s)", x, y);
    }
    public double distanceTo(Point q){
        return Math.hypot(x - q.x, y - q.y);
    }
    public static void main(String[] args) {
        Point p = new Point(3,4);
        Point q = new Point(0,0);
        System.out.println(p.distanceTo(q));
    }
}